Item.cpp

```diff
@@ -0,0 +1,7 @@
+#include "Item.h"
+
+Item::Item
+(
+    const std::string& name_, 
+    int effect 
+) : name(name_), boost(effect) { }
```

Comment:
- [ ] REVISE

Reply:
You still haven't fixed this ugly code style.
- [ ] REVISE

Reply:
ok done.